packages/components/src/internal/components/samples/SamplesAddButton.tsx

```diff
@@ -26,12 +27,23 @@ export const SamplesAddButton: FC<Props> = memo(props => {
     // Issue 43113: If a queryInfo is associated with this create action then respect its settings for display
     if (!showInsertNewButton && !showImportDataButton) return null;
 
+    const items = (
+        <>
+            {showInsertNewButton && <MenuItem href={createSampleHref}>Add Manually</MenuItem>}
+            {showImportDataButton && !hideImport && <MenuItem href={importSampleHref}>Import from File</MenuItem>}
+        </>
+    )
     return (
         <RequiresPermission permissionCheck="any" perms={PermissionTypes.Insert}>
+            {!asSubMenu && (
             <DropdownButton title="Add" id="samples-add-menu" bsStyle="success">
-                {showInsertNewButton && <MenuItem href={createSampleHref}>Add Manually</MenuItem>}
-                {showImportDataButton && !hideImport && <MenuItem href={importSampleHref}>Import from File</MenuItem>}
+                {items}
             </DropdownButton>
+            )}
+            {asSubMenu && (
+                <SubMenuItem text="Create Samples">{items}</SubMenuItem>
```

Comment:
there will be a small change here to add the `inline` prop after this and my current feature branches get merged together, just an FYI for whichever of us goes second.